ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.cy.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import ComponentTestProviders from 'test-utils/ComponentProviders';
+
+import CompoundSearchFilter from './CompoundSearchFilter';
+import {
+    clusterSearchFilterConfig,
+    deploymentSearchFilterConfig,
+    imageSearchFilterConfig,
+} from '../types';
+
+const selectors = {
+    selectToggle: 'button[aria-label="compound search filter entity selector toggle"]',
+    selectItems: 'ul[aria-label="compound search filter entity selector items"] li',
+};
+
+function Wrapper({ config }) {
+    return <CompoundSearchFilter config={config} />;
+}
+
+function setup(config) {
+    cy.mount(
+        <ComponentTestProviders>
+            <Wrapper config={config} />
+        </ComponentTestProviders>
```

Comment:
Depending on what the final component looks like, I think you can do without these TestProvider wrappers. Right now I believe they are only necessary if you need any of:
- GraphQL mocks
- Redux selectors
- Interaction with page routing

Reply:
We'll probably end up needing to mock GraphQL once we get to autocomplete. I'll leave it for now.